src/config-local.tsx

```diff
@@ -17,8 +17,18 @@ export default {
   // Fetch chain data interval
   REFRESH_INTERVAL_MILLISECONDS: 10000,
 
+  // Health check data refresh interval
+  HEALTH_CHECK_POLL_INTERVAL_MILLISECONDS: 120_000,
```

Comment:
is this every 2 minutes? we should do this like every 15 seconds rathe?

Reply:
It takes a while to get the last message times for nonces from the parachain. This is because we have to walk the blocks and look for nonce updates. It runs approximately 30 seconds on staging. So that's why I have it set for 2 minutes. We can do 30 seconds but it would mean the health check code constantly running in the background. There are some optimisations we can potentially do to speed it up like binary search through the blocks as both block numbers and nonces increment linearly.